learn/getting_started/quick_start.md

```diff
@@ -92,10 +92,17 @@ If the Rust toolchain is already installed, clone the repository on your local s
 
 ```bash
 git clone https://github.com/meilisearch/Meilisearch
```

Comment:
GitHub links are case-insensitive. However the folder that is generated when you run `git clone` will be based on the last piece of text in the link, e.g. the above code generates a folder called `Meilisearch`.

As @guimachiavelli mentioned recently, certain Unix-based OS can complain if you have a folder called `Meilisearch` and try to `cd meilisearch`, because they are case-sensitive. So we have to be careful about this. As a good rule of thumb, the URL should always be all-lowercase, and the folder you `cd` to as well.

```suggestion
git clone https://github.com/meilisearch/meilisearch
```